authui/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
-  "plugins": ["@oursky", "compat"],
+  "plugins": [
```

Comment:
Please do not make any changes to this files. If you need to access globals, you can do it in a per-file basic. https://eslint.org/docs/latest/use/configure/language-options#using-configuration-comments